config/routes.rb

```diff
@@ -240,7 +240,8 @@
         end
       end
 
-      resources :wikis, only: [:show, :edit, :destroy, :create], constraints: {id: /[a-zA-Z.0-9_\-\/]+/} do
+      resources :wikis, only: [:show, :edit, :destroy, :create],
+      constraints: {id: /\S+/} do
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.